change/@microsoft-teams-js-6364da81-f8f2-4864-b4b8-305ef45c6548.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "minor",
+  "comment": {
+    "title": "new API to support show/hide of app share button",
```

Comment:
The comment should follow our comment guidelines in CONTRIBUTION.md. Should be something like "Added support for showing and hiding the app share button to the `meeting` capability".